src/components/List/MessageConversationList.js

```diff
@@ -98,9 +98,9 @@ class MessageConversationList extends Component {
                       ))
                     : !this.props.selectedMessageType.loading && (
                           <Subheader>
-                              {i18n.t(
-                                  `No ${messageType.displayName.toLowerCase()} messages`
-                              )}
+                              {i18n.t(`No messages in '{{messageType}}'`, {
```

Comment:
[TRIVIAL] Are we sure we want to use single quotes for the presentation string ?